Cython/Compiler/ModuleNode.py

```diff
@@ -1628,6 +1629,27 @@ def generate_new_function(self, scope, code, cclass_entry):
         code.putln(
             "}")
 
+    def generate_del_function(self, scope, code):
+        tp_slot = TypeSlots.get_slot_by_name("tp_finalize")
```

Comment:
```suggestion
        tp_slot = TypeSlots.get_slot_by_name("tp_finalize", scope.directives)
```

(I think).

Reply:
Right, that changed in master. Looks reasonable.